Sample Data/Feeds/EventLogManagement.csv

```diff
@@ -1,90 +1,90 @@
-An activity log alert should exist for Delete Network Security Group,Service Logging
-An activity log alert should exist for Delete Security Solution,Service Logging
-Diagnostic logs in Kubernetes services should be enabled,Service Logging
-An activity log alert should exist for Delete SQL Server Firewall Rule,Service Logging
-An activity log alert should exist for Delete Network Security Solution,Service Logging
-An activity log alert should exist for specific Policy operations,Service Logging
+RecommendationDisplayName,LoggingCategory,MaturityLevel
+Log Analytics agent should be installed on your virtual machine,Collection Agent,Event Logging (EL0)
+Guest Configuration extension should be installed on your machines,Collection Agent,Event Logging (EL0)
+Deploy the Windows Guest Configuration extension to enable Guest Configuration assignments on Windows VMs,Collection Agent,Event Logging (EL0)
+Network traffic data collection agent should be installed on Windows virtual machines,Collection Agent,Event Logging (EL0)
+Network traffic data collection agent should be installed on Linux virtual machines,Collection Agent,Event Logging (EL0)
+Virtual machines should have the Log Analytics extension installed,Collection Agent,Event Logging (EL0)
+Deploy the Linux Guest Configuration extension to enable Guest Configuration assignments on Linux VMs,Collection Agent,Event Logging (EL0)
+The Log Analytics agent should be installed on virtual machines,Collection Agent,Event Logging (EL0)
+Auto provisioning of the Log Analytics agent should be enabled on your subscription,Collection Agent,Event Logging (EL0)
+Log Analytics agent should be installed on your Linux-based Azure Arc machines,Collection Agent,Event Logging (EL0)
+Install endpoint protection solution on your machines,Collection Agent,Basic Event Logging (EL1)
+Install endpoint protection solution on virtual machines,Collection Agent,Basic Event Logging (EL1)
+Windows Defender Exploit Guard should be enabled on your machines,Collection Agent,Basic Event Logging (EL1)
+Endpoint protection should be installed on your machines,Collection Agent,Basic Event Logging (EL1)
+Microsoft Antimalware for Azure should be configured to automatically update protection signatures,Collection Agent,Intermediate Event Logging (EL2)
+Microsoft IaaSAntimalware extension should be deployed on Windows servers,Collection Agent,Basic Event Logging (EL1)
+Endpoint protection health issues should be resolved on your machines,Log Collection Health & Retention,Basic Event Logging (EL1)
+Log Analytics agent health issues should be resolved on your machines,Log Collection Health & Retention,Basic Event Logging (EL1)
+Audit retention for SQL servers should be set to at least 90 days,Log Collection Health & Retention,Basic Event Logging (EL)
+Saved-queries in Azure Monitor should be saved in customer storage account for logs encryption,Event Logging (EL0)
```

Comment:
Github flagging this row only has 2 fields instead 3 so csv is not displaying correctly.